bodhi/tests/server/services/test_graphql.py

```diff
@@ -15,12 +15,47 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+from graphene.test import Client
+
 from bodhi.tests.server import base
+from bodhi.server.services.graphql import schema
+from bodhi.server.models import PackageManager, Release
 
 
 class TestGraphQLService(base.BasePyTestCase):
     """This class contains tests for a /graphql endpoint"""
     def test_get(self):
-        """Ensure that "Hello World" is returned"""
-        res = self.app.get('/graphql')
-        assert res.body == b'"Hello World"'
+        """Ensure that a GraphQL response is returned"""
+        res = self.app.get('/graphql?query={%0A%20 allReleases{%0A%20%20%20 name%0A%20 }%0A}')
+        assert res.body == b'{"data":{"allReleases":[{"name":"F17"}]}}'
+
+    def test_allReleases(self):
+        """Testing allReleases"""
+        client = Client(schema)
+        release = Release(
+            name='F22', long_name='Fedora 22',
+            id_prefix='FEDORA', version='22',
+            dist_tag='f22', stable_tag='f22-updates',
+            testing_tag='f22-updates-testing',
+            candidate_tag='f22-updates-candidate',
+            pending_signing_tag='f22-updates-testing-signing',
+            pending_testing_tag='f22-updates-testing-pending',
+            pending_stable_tag='f22-updates-pending',
+            override_tag='f22-override',
+            branch='f22',
+            package_manager=PackageManager.dnf,
+            testing_repository='updates-testing')
+
+        self.db.add(release)
+        self.db.commit()
+
+        executed = client.execute("""{  allReleases{  name  }}""")
```

Comment:
It would be nice to test for some of the trickier fields like `state` or `package_manager` (or both!). You could request it from allReleases and make sure it matched the value provided to db.add(). I think that could be a separate test.